README.md

```diff
@@ -108,6 +108,50 @@ HTML styled "just right".
 Jekyll will print a local URL where the site can be viewed (usually,
 [http://0.0.0.0:4000/](http://0.0.0.0:4000/)).
 
+### Testing using Docker environment 
+
+#### Build environment
+A containerized development environment can be built using the local
+Dockerfile.
+
+Run the build-images.sh script to generate the development environment and
+associated images.
+
+```bash
+docker build -t webdev .
+```
+
+This action will produce a `webdev` container which will execute a
+`jekyll serve` command with the polling option enabled.
+
+The webdev container does not store any site content inside of it.
+For rendering to function, the local directory must be mounted inside
+the container in a read/write configuration.
+
+This mounting is accomplished with the volume `-v` flag in the
+following `docker run` command.
```

Comment:
When you run a container using the webdev image, your current working
directory will be mounted, so that any changes made by the build inside
the container will be reflected in your local workspace. This is done with the
`-v` flag. To run the container, execute the following command: